usaspending_api/references/v2/views/tas_autocomplete.py

```diff
@@ -77,7 +82,70 @@ def _business_logic(filters, requested_component, limit):
         return {"results": results}
 
     @cache_response()
-    def post(self, request, requested_component):
+    def post(self, request):
         request_data = self._parse_and_validate_request(request.data)
-        results = self._business_logic(request_data.get("filters", {}), requested_component, request_data["limit"])
+        results = self._business_logic(request_data.get("filters", {}), request_data["limit"])
         return Response(results)
+
+
+class TASATAAutocomplete(TASAutocomplete):
```

Comment:
For the borderline dyslexics in the room, these class names are very difficult to read